webapp/channels/src/i18n/en.json

```diff
@@ -1412,7 +1412,13 @@
   "admin.license.enterpriseEdition.subtitle": "This is an Enterprise Edition for the Mattermost {skuName} plan",
   "admin.license.enterprisePlanSubtitle": "We’re here to work with you and your needs. Contact us today to get more seats on your plan.",
   "admin.license.enterprisePlanTitle": "Need to increase your headcount?",
-  "admin.license.freeEdition.subtitle": "Purchase Professional or Enterprise to unlock enterprise features.",
+  "admin.license.enterpriseToAdvancedAdvantage.attributeBasedAccess": "Attribute-based access control",
+  "admin.license.enterpriseToAdvancedAdvantage.channelWarningBanners": "Channel warning banners",
+  "admin.license.enterpriseToAdvancedAdvantage.adLdapGroupSync": "AD/LDAP group sync",
+  "admin.license.enterpriseToAdvancedAdvantage.advancedWorkflows": "Advanced workflows with Playbooks",
+  "admin.license.enterpriseToAdvancedAdvantage.highAvailability": "High availability",
+  "admin.license.enterpriseToAdvancedAdvantage.advancedCompliance": "Advanced compliance",
+  "admin.license.freeEdition.subtitle": "This is an unlicensed server and does not have commercial support. Purchase a Professional, Enterprise, or Enterprise Advanced plan to unlock paid features.",
```

Comment:
@harshilsharma63 @pvev can you guys remind me what else I need to do after I added all these new strings here to get the CI happy? 
<img width="894" alt="Screenshot 2025-05-06 at 10 26 48 AM" src="https://github.com/user-attachments/assets/8544014e-682f-424e-960e-a65fbd45368f" />